frontend/package.json

```diff
@@ -43,5 +43,8 @@
     "vite-plugin-vue-devtools": "^7.5.4",
     "vitest": "^2.1.4",
     "vue-tsc": "^2.1.10"
+  },
+  "optionalDependencies": {
+    "@rollup/rollup-linux-arm64-musl": "^4.27.3"
```

Comment:
added this to resolve this issue: [github issue](https://github.com/vitejs/vite/discussions/15532)

Reply:
https://sandipdulal.medium.com/fixing-vite-build-error-on-linux-and-windows-using-docker-error-cannot-find-module-e73bb2fb479d